modules/mri_violations/php/NDB_Menu_Filter_Form_mri_violations.class.inc

```diff
@@ -25,6 +25,7 @@
  */      
 class NDB_Menu_Filter_Form_mri_violations extends NDB_Menu_Filter_Form
 {
+    var $AjaxModule = true;
```

Comment:
I believe `var` keyword should not be used, as it is was replaced with `public, private, etc` since PHP5. The parent classes use `var` all over the place, so I'm not sure if we can/should update child classes to new syntax

@driusan Thoughts?